pkg/backends/options/options_test.go

```diff
@@ -269,25 +269,11 @@ func TestValidate(t *testing.T) {
 		sw       []intSwapper
 		expected any
 	}{
-		{ // case 0 - MaxShardSizeMS > 0 and MaxShardSizePoints > 0 are mutually exclusive
```

Comment:
Double checking on the removal of this test case. the intent is to error if both `MaxShardSizeMS` and `MaxShardSizePoints` are both non-zero (only one can be, which determines the pruning strategy to use).  In removing MS, `MaxShardSizeMS` might be better named `MaxShardSizeTime` to be better distinguished from `Points`

Reply:
the specific issue here is the types are no longer both ints, so the test structure doesn't quite fit

I'll add a more specific test for this

Reply:
added a new test